duckduckgo/duckduckgo-autofill
# Fix minor regression

**Reviewer:** @alistairjcbrown 
**Asana:** https://app.asana.com/0/0/1204595906186359/f

## Description
This fixes a minor use case of tabbing from field to field when the form overflows the viewport.

## Steps to test
Added test case and verified on device. Steps to test:

1. Visit https://www.roboform.com/filling-test-all-fields and make sure you have an identity with all data types
2. Shrink your window so that some fields are below the fold
3. Tab from field to field
4. The autofill pulldown should open when tabbing to fields below the fold

![CleanShot 2023-05-16 at 10 28 51](https://github.com/duckduckgo/duckduckgo-autofill/assets/1828326/6900b9ce-89bd-440b-8189-4a1616e4d96d)
